test/runnable/test42.d

```diff
@@ -65,7 +62,7 @@ void test3()
 {
     auto i = mixin("__LINE__");
     printf("%d\n", i);
-    assert(i == 66);
+    assert(i == 65);
```

Comment:
That should be 63, shouldn't it ?

Reply:
Generally, when relying on `__LINE__` being a specific number, it is best to precede it with a `#line` directive, so it won't constantly break.

Reply:
@Geod24 Yup. Fixed.